main/tests/test_feeds.py

```diff
@@ -35,6 +35,26 @@ def test_rss_feed(self):
             f"//{self.user.username}.{settings.CANONICAL_HOST}/blog/{self.data['slug']}/</link>",
         )
 
+    def test_rss_feed_with_custom_domain(self):
+        self.user.custom_domain = "mataroa-blog.com"
+        self.user.save()
+
+        response = self.client.get(
+            reverse("rss_feed"),
+            HTTP_HOST=self.user.custom_domain,
+        )
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response["Content-Type"], "application/rss+xml; charset=utf-8")
+        self.assertContains(response, self.data["title"])
+        self.assertContains(response, self.data["slug"])
+        self.assertContains(response, self.data["body"])
+        self.assertContains(
+            response,
+            f"//{self.user.custom_domain}/blog/{self.data['slug']}/</link>",
```

Comment:
@sirodoht to be fair, we might as well just use `user.blog_url` eveywhere in the code. What do you think?

Reply:
hmm yeah, I wonder why it's not used. maybe I added `blog_url` later at some point